src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -57,6 +56,18 @@ public static partial class PlatformDetection
         public static bool IsOpenSsl3_4 => IsOpenSslVersionAtLeast(s_openssl3_4Version);
         public static bool IsOpenSsl3_5 => IsOpenSslVersionAtLeast(s_openssl3_5Version);
 
+        private static readonly Lazy<bool> s_IsSymCryptOpenSsl = new(() =>
+        {
+            return IsAzureLinux &&
```

Comment:
We can relax this to just `IsLinux` but to my knowledge there is no case today where SCOSSL is used outside of Azure Linux.